test/unit/controllers/declaration/DispatchLocationControllerSpec.scala

```diff
@@ -62,7 +62,7 @@ class DispatchLocationControllerSpec extends ControllerSpec {
 
         withNewCaching(aDeclaration(withDispatchLocation(OutsideEU)))
 
-        val result = controller.displayPage(Mode.Normal)(getRequest())
+        private val result = controller.displayPage(Mode.Normal)(getRequest())
```

Comment:
There is no point of making this result private. This is inside the test suite so this value cannot be accessed outside.

This comment to the whole file

Reply:
I made a few things private because IntelliJ had warnings about them (when it does the warning check just before doing a commit).  I normally ignore the warnings but on my new, super-fast machine I thought I'd take a look.

I'm not going to reverse these changes now that I have a precious approval.  :-)